core/src/commonMain/kotlin/com/ustadmobile/core/view/ContentEntryEdit2View.kt

```diff
@@ -22,6 +22,12 @@ interface ContentEntryEdit2View: UstadEditView<ContentEntryWithLanguage>{
 
     var entryMetaData: ImportedContentEntryMetaData?
 
+    var entryCompressed: Boolean
+
+    var videoDimensions: Pair<Int, Int>
+
+    var videoFilePath: String?
```

Comment:
This is not actually a filepath! It is a URL (as it is also used to represent an http link for video streaming). Can you please rename accordingly?

Reply:
renamed to VideoUri